maturity-level-two/README.md

```diff
@@ -11,3 +11,42 @@ You should:
 - Add your OpenAPI specs to source control given this is part of your application
 - Validate changes to your OpenAPI specs to avoid specification violations ([user guide](docs/validating-open-api-specs.md))
 - Unit test Open API validation to automatically detect breaking changes
+
+ ## Content negotiation
+ With content negotiation we detremine in which format we'd like to receive requests and in which format we'd like to return our responses. For REST API's common request formats are JSON and XML, the most common response formats are JSON, XML and JSON. When someone does an http request to your api he should use the 'Content-Type' header to specify in which format the request is done and use the 'Accept' header to determine in which format he expects a response. Depending on these headers your api will decide in which format the request is send in in which format it will return it's response. When a format is not supported you should return the 406 error response code (not acceptable). When no 'Accept' header is defined a default response format should be used. Becaus of it's lightness and fastness, JSON is getting the standard the last years. But in certain situations other formats still have their advantages.
+
+ You should use the right attributes in order to make sure your API is documented correctly. You can use the [Consumes] or [Produces] attribute (or both, depending on your method) in order to specify what content type your api consumes and produces. You can optionally use these attributes in on your controller class if you'd like to define this attribute for each method of your controller. In order to clarify the response body scheme of a request, you can use the [SwaggerResponse] attribute when you're using Swashbuckle. Make sure you use all parameters in the [SwaggerResponse] attribute when doing an HTTP GET with response status OK, the type argument might not be needed in other cases. 
```

Comment:
Improved readability a bit:
```suggestion
REST APIs should provide documentation on the supported formats in their OpenAPI specifications. This allows API consumers to get an understanding of the formats they can use and/or expect.

You can use the [Consumes] or [Produces] attribute (or both, depending on your method) in order to specify what content type your API consumes and produces. This can be applied on a controller or operation level, depending on your scenario. 
```

> In order to clarify the response body scheme of a request, you can use the [SwaggerResponse] attribute when you're using Swashbuckle. Make sure you use all parameters in the [SwaggerResponse] attribute when doing an HTTP GET with response status OK, the type argument might not be needed in other cases.

This is certainly the case and agree, but this should be added here instead: https://github.com/CoditEU/practical-api-guidelines/tree/master/maturity-level-one#generating-openapi-documentation

Would you mind moving this to a seperate PR along with the sample updates you did please?